shell/init.go

```diff
@@ -12,6 +12,7 @@ antibody() {
 	case "$1" in
 	bundle|update)
 		while read -u 3 bundle; do
+			touch /tmp/antibody-log && chmod 777 /tmp/antibody-log
```

Comment:
this will repeat for each entry on the `while` command, maybe put it before?

also: will this work if the file already exists? Maybe just put the logs somewhere else will be good enough... (like `$TMPDIR/antibody.log`)